Cargo.toml

```diff
@@ -44,6 +44,7 @@ tokio-stream = "0.1"
 nix = "0.25.0"
 libc = "0.2"
 static_assertions = "1.1.0"
+bytes = "1.2.1"
```

Comment:
`1.2` is better. This will allow updates that do not break backwards compatibility